### ISSUE TITLE
tensorflow-1-public/C1/W2/ungraded_labs/C1_W2_Lab_1_beyond_hello_world.ipynb Exercise 7

### ISSUE THREAD

**Issue by gladwig2** [https-deeplearning-ai/tensorflow-1-public#8]
![image](https://user-images.githubusercontent.com/8951837/129573940-2b46410a-1c87-4a34-bfd1-08280a08fbb0.png)

Am I missing something? The exercise describes trying to fit without normalizing, but the Test/Training data is still divided by 255?

**Comment by cfav-dev** [comment]
I looked at the notebook and it was already revised previously. They added comments to the lines that can be removed. Thanks!
